src/server/services/procedures/mortality/mortality.js

```diff
@@ -0,0 +1,228 @@
+/**
+ * This service accesses data from the human mortality database,
+ * which tabulates death rates broken down by age group and gender
+ * for various countries.
+ * 
+ * @alpha
+ * @service
+ * @category Health
+ */
+'use strict';
+
+// source: https://www.mortality.org/
```

Comment:
It would be good to put this url in the service's help message. I usually just add a line by itself and say something like "For more information, check out <url>"